bookkeeper-server/src/main/java/org/apache/bookkeeper/client/PendingReadOp.java

```diff
@@ -389,6 +390,7 @@ synchronized BookieSocketAddress sendNextRead() {
                 return null;
             }
 
+            // ToDo: pick replica with writable PCBC
```

Comment:
if the TODO is not implemented, please create an issue here and add the github issue at the TODO comment.

Reply:
will do. https://github.com/apache/bookkeeper/issues/1239